server/routing/lobby_test.go

```diff
@@ -140,6 +140,10 @@ func TestLobbyAPI(t *testing.T) {
 		t.Fatal("expected initial game state to have something in it.")
 	}
 
+	if !assert.NotEqual(t,newGameMsg.GameState.Players[0].Deck,newGameMsg.GameState.Players[1].Deck) {
```

Comment:
Do you know if this equality check would actually fail if they were shuffled the same way? (If you take the `.UnixNano()` out, does this test fail?

Reply:
(I am suspicious of an equality check directly against `.Deck`)